homeassistant/components/airly/model.py

```diff
@@ -1,15 +1,14 @@
 """Type definitions for Airly integration."""
 from __future__ import annotations
 
-from typing import Callable, TypedDict
+from dataclasses import dataclass
+from typing import Callable
 
+from homeassistant.components.sensor import SensorEntityDescription
```

Comment:
It's better to keep all the sensor integration imports in the sensor platform if possible.

Reply:
OK, I will move `AirlySensorEntityDescription` class to sensor platform.